### ISSUE TITLE
Migrate 311 > Knack integration to PostgREST

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#7405]
TLDR: Replace the current 311 > Knack-Proxy integration with a central PostgREST datastore.

We have been thinking internally about a better way to manage our ESB integration. As you know, we have periodic outages for which we haven't been able to identify a solution. We experienced one of these outages over the weekend, in fact.

We're also thinking about how we can more efficiently scale our integration to incorporate additional Knack applications as well as manage internal reporting needs.

The solution we've zeroed in on is to have a centralized PostgRESTdata store of 311 SRs that sits between our applications and the ESB. This will address two main issues.

### Outages
The ESB will interface directly with our own database API. We suspect that the current outages stem from latency issues between the ESB, our proxy service, and the Knack API. We unfortunately have very little control/visibility of the Knack API.

Under this new scheme, the ESB can make a POST request directly to the data store. The database technology we have in mind is extremely resilient and fast—we currently use it for thousands of transactions/day.

### Scalability
This intermediary data store will allow us to make modifications to our Knack apps, and add new ones, without needing modify the ESB integration. You will only need to maintain one set of mappings, tokens, etc. for all of our integration needs.

### Todo

- [ ] Meet with ESB team to discuss changes
- [ ] Setup PostgREST tables
- [ ] Write script to inegrate PostgREST and Knack (we can use atd-knack-services)
- [ ] Test/deploy

![Current state](https://user-images.githubusercontent.com/14793120/136824770-ffe3e711-82ce-4cec-8772-ce7c499db67d.jpg)
![Future state](https://user-images.githubusercontent.com/14793120/136824774-6fd6d84c-0b99-4051-824b-cc504b1b3102.jpg)
